docs/user-guide/editor.md

```diff
@@ -173,15 +173,17 @@ If a `setup` editor is present, that's the only PyEditor that needs a config.
 Any subsequent related editor will reuse the config parsed and bootstrapped for
 the `setup` editor.
 
-## Execution via keyboard
+## Run via keyboard
 
-Accordingly with the Operating system you are using, a combination of either `Ctrl-Enter`, `Cmd-Enter` or `Shift-Enter` would implicitly execute the code and show results within the `target` output node without ever needing to reach the *Run* button and lose the focus.
+Depending on your operating system, a combination of either `Ctrl-Enter`,
+`Cmd-Enter` or `Shift-Enter` will execute the code in the editor (no need to
+move the mouse to click the run button).
 
-## Execution Override
+## Execution override
```

Comment:
as you changed previous *Execution via keyword* to *Run* ... shouldn't this be then *Run override* too?

Reply:
Good point. Fixed.